indexing-service/src/main/java/org/apache/druid/indexing/overlord/hrtr/HttpRemoteTaskRunner.java

```diff
@@ -1311,76 +1246,87 @@ private void pendingTasksExecutionLoop()
              .addData("taskId", taskId)
              .emit();
 
-          // taskComplete(..) must be called outside of statusLock, see comments on method.
+          // taskComplete(..) must be called outside workerStatusLock, see comments on method.
```

Comment:
Let's get rid of this comment, and convert the comment on the `taskComplete()` method into a javadoc.